frontend/components/GamesList.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
You mention code duplication on Discord. There is indeed some duplicated code here and while it would be possible to refactor this to a state where you have the common code in one place I am not sure I'd bother with that.

You could have something a generic component for filterable lists, but then you'd have to move the logic of preparing the initial list outside of the component. You could do a [high-order-component](https://reactjs.org/docs/higher-order-components.html) that just adds the filtering too. 

For now let's just go with what we have and if you want to refactor this you can do it later. Preferably you'd do it once the abilities list page is done, because that may change how you view things.